<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArithSettings</class>
 <widget class="QDialog" name="ArithSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QComboBox" name="mulBox">
     <property name="font">
      <font>
       <family>Ark Pixel 12px Proportional zh_cn</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>20以内</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200以内</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2000以内</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20000以内</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="subBox">
     <property name="font">
      <font>
       <family>Ark Pixel 12px Proportional zh_cn</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>20以内</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200以内</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2000以内</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20000以内</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="divCheck">
     <property name="font">
      <font>
       <family>Ark Pixel 12px Proportional zh_cn</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>是否启用除法</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="negativeCheck">
     <property name="font">
      <font>
       <family>Ark Pixel 12px Proportional zh_cn</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>是否包含负数</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="divBox">
     <property name="font">
      <font>
       <family>Ark Pixel 12px Proportional zh_cn</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>20以内</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200以内</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2000以内</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20000以内</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <family>Ark Pixel 12px Proportional zh_cn</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="mulCheck">
     <property name="font">
      <font>
       <family>Ark Pixel 12px Proportional zh_cn</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>是否启用乘法</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="addCheck">
     <property name="font">
      <font>
       <family>Ark Pixel 12px Proportional zh_cn</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>是否启用加法</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="subCheck">
     <property name="font">
      <font>
       <family>Ark Pixel 12px Proportional zh_cn</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>是否启用减法</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="addBox">
     <property name="font">
      <font>
       <family>Ark Pixel 12px Proportional zh_cn</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>20以内</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200以内</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2000以内</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20000以内</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ArithSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ArithSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
